google/ads/google_ads/config.py

```diff
@@ -129,6 +129,25 @@ def load_from_yaml_file(path=None):
 
     return parse_yaml_document_to_dict(yaml_doc)
 
+@_config_validation_decorator
+@_config_parser_decorator
+def load_from_dict(config_dict):
+    """ Check if the argument is dictionary or not
+
+    Args:
+        Configuration dictonary of credentials
```

Comment:
You'll want to name the parameter you're referring to, see [here](https://github.com/googleads/google-ads-python/blob/master/google/ads/google_ads/client.py#L49).